src/SixLabors.Core/SixLabors.Core.csproj

```diff
@@ -34,24 +25,28 @@
         <CodeAnalysisRuleSet>..\SixLabors.ruleset</CodeAnalysisRuleSet>
     </PropertyGroup>
 
+    <PropertyGroup Condition=" $(TargetFramework.StartsWith('netcoreapp2')) ">
```

Comment:
`$(TargetFramework.StartsWith('netcoreapp') && !TargetFramework.StartsWith('netcoreapp1'))`?


That should protect us for NET Core 3+

Reply:
Let's wait until we figure out what new targets we'll be adding to start utilizing intrinsics before we try to future proof this.  NETSTANDARD2.1 is also coming.

Reply:
Yeah ok. Let's wait til we need to.